src/composables/useNumbers.ts

```diff
@@ -14,7 +14,8 @@ interface Options {
 export interface FNumOptions extends Intl.NumberFormatOptions {
   fixedFormat?: boolean; // If true, don't auto-adjust based on number magnitde
   abbreviate?: boolean; // If true, reduce number size and add k/M/B to end
-  dontAdjustLarge?: boolean; // If true, don't auto-adjust if the number is large
+  dontAdjustLarge?: boolean; // If true, don't auto-adjust if the number is large,
+  fractionDigits?: number; // set fraction digits
```

Comment:
You can just pass `maximumFractionDigits` into fNum2 instead of adding this, as it extends `Intl.NumberFormatOptions`

Reply:
fixed